resources/js/tags.js

```diff
@@ -1,5 +1,41 @@
 $(document).ready(function() {
     window.initTags = function() {
+        $.each($('.position-dropdown'), function () {
```

Comment:
Since the function is called initTags, let's have this foreach come after the already existing one  for clarity :)